src/adcm_pytest_plugin/docker_utils.py

```diff
@@ -372,7 +372,7 @@ def upgrade(self, target: Tuple[str, str]) -> None:
         with allure.step("Start newer adcm"):
             dw = DockerWrapper(base_url=self.container_config.docker_url)
             container, _ = dw.adcm_container_from_config(self.container_config)
-            _wait_for_adcm_container_init(container, self.ip, self.port, timeout=30)
+            _wait_for_adcm_container_init(container, self.ip, self.port)
```

Comment:
Why?

Reply:
Default is 120 sec 30 is not enough for DB migration